### ISSUE TITLE
Expand modo --help message to show all

### ISSUE THREAD

**Issue by almutlue** [sdsc-ordes/modo-api#34]
`modo --help` displays only part of the description for some commands (see publish and remove below).
Is this related to some typer setting and can we change this?

```
(modo-py3.10) (base) stefan@ordesmain:~/modo-api$ modo --help
Usage: modo [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  add      Add elements to a digital object.
  create   Create a digital object.
  publish  Creates a semantic artifact allowing to publish a digital...
  remove   Removes the target element from the digital object, along with...
  show     Show the contents of a digital object.
```

**Comment by almutlue** [comment]
Instead shorten the sentences/docstrings

**Comment by cmdoret** [comment]
Fixed in 7404316
